content/updates/2019-07-12-VPN-backhaul-drupal-8-and-new-team-members.md

```diff
@@ -0,0 +1,89 @@
+
+Our team has successfully completed this year’s re-authorization. Current and prospective customers can view our audit materials in the FedRAMP document repository on MAX.gov by [requesting access from FedRAMP](https://cloud.gov/overview/security/fedramp-tracker/#start-the-ato-process).
+
+## New additions
+
+
+### Try an experimental feature for no-downtime deployments
+
+It’s often a challenge to build a process for deploying new versions of your applications with zero downtime, without needing to double your memory usage. The [latest version of the Cloud Foundry command-line interface (CLI)](https://github.com/cloudfoundry/cli/releases) includes new beta commands to push apps using [rolling deployment](https://docs.cloudfoundry.org/devguide/deploy-apps/rolling-deploy.html). This makes it much easier to implement [blue-green deployment patterns](https://docs.cloudfoundry.org/devguide/deploy-apps/blue-green.html) without doubling your memory quota. (For example, this can replace the unmaintained [autopilot plugin](https://github.com/contraband/autopilot).) Install [the CF CLI update](https://github.com/cloudfoundry/cli/releases), try it out, and let us know if you have any questions!
+
+### Functions support for MySQL databases
+
+If you need to set up a MySQL schema that includes functions, procedures, and triggers, you can now create new databases that have that feature enabled. [To do this, see the “Options” documentation for RDS databases.](https://cloud.gov/docs/services/relational-database/#options) This allows you to build a wider range of applications on cloud.gov.
+
+### Improved instructions for running Drupal 8 for managing content
+
+A lot of government teams run Drupal for content management for their websites, including the [Drupal 8](https://www.drupal.org/8) version. With some modifications, Drupal runs well on cloud.gov. We’ve improved our [instructions and sample code](https://github.com/18F/cf-ex-drupal8) showing how to run Drupal in cloud.gov with best practices including configuration-as-code and continuous deployment principles. Contact our team if you need additional help!
+
+### Automatically label your applications with version control information
+
+Do you want to track which version of your code is running in each of your deployed applications? You can now use the cloud.gov API to automatically label your applications and other resources with your choice of metadata, such as the relevant git commit hash (SHA). For instructions, see [this guide to resource labeling](https://www.cloudfoundry.org/blog/labeling-cloud-foundry-api-resources-with-a-git-sha/).
+
+## Updated
+
+We encourage staying up to date with the latest Cloud Foundry Command-Line Interface (CLI) versions. The latest version is v6.45.0. [You can download the CF CLI update here.](https://github.com/cloudfoundry/cli/releases)
```

Comment:
The latest version is actually v6.46.0 now! Encourage leaving that number out or updating it.